README.md

```diff
@@ -7,78 +7,75 @@
 <h1 align="center">M4ag Firmware</h1>
 
 [![Compile test](https://github.com/Mesh4all/m4a-firmware/actions/workflows/compilation_check.yml/badge.svg)](https://github.com/Mesh4all/m4a-firmware/actions/workflows/compilation_check.yml)
+[![License](https://img.shields.io/badge/license-Apache_License_2.0-blue)](http://www.apache.org/licenses/LICENSE-2.0)
 
 # m4a-firmware
-Firmware for all available devices.
 
-**Important Notice**
+<p style='text-align: justify'> The firmware for connect everyone and everything in this world.
+Expand all your wireless network limits and join to the big mesh.</p> 
 
-> This README is a work in progress (WIP), some sections may be incomplete or need some extra work (TODO)
+<p style='text-align: justify'>m4a-firmware is a software implemented in hardware able to provide enroute data between devices. The principal proposite is create only a wide wireless network.
+m4a-firmware is based under RIOT OS, and we follow some of its design principles: energy-efficiency and modularity. We implemented our network vision approach and expanded capabilities to join some devices to networks.</p>
```

Comment:
RIOT OS -> RIOT-OS
add please link to the RIOT-OS's repo it's the first  mention